plugins/catalog/src/alpha.tsx

```diff
@@ -65,12 +81,191 @@ export const CatalogSearchResultListItemExtension =
       ),
   });
 
+/** @alpha */
+export function createCatalogFilterExtension<
```

Comment:
@Rugvip , @freben should I extract this to the `@backstage/plugin-catalog-react`? So other plugins, like `TechDocs could use this function for exporting their specific filters.

Reply:
So far I think it's only been for creating custom filters in apps right? Let's not make the change unless it's already present in the existing system imo, but we could consider this in the future.